mautrix_signal/portal.py

```diff
@@ -386,6 +390,7 @@ async def _handle_matrix_message(
                 quote=quote,
                 attachments=attachments,
                 timestamp=request_id,
+                req_id=signald_req_id,
```

Comment:
This looks like it'll use the same request ID for each retry, which feels wrong, but I guess it's fine if the retries are only for signald going down and/or if signald doesn't care about request ID uniqueness

Reply:
Oh, good point.